openedx/stanford/common/djangoapps/enrollment/urls.py

```diff
@@ -1,15 +1,18 @@
 """
 URL for enrollment API
 """
+from django.conf import settings
 from django.conf.urls import patterns, url
 
-from .views import UpdateEnrollmentView
+from .views import EnrollmentStatusView
 
 urlpatterns = patterns(
     'enrollment.views',
     url(
-        r'^update_status$',
```

Comment:
The URL doesn't need to mention the action (update),
that is implied via the HTTP method (POST).
    
Instead, we can abstract this a bit and use it again in the future.
The other fork we had of `get_roster` could be expressed as a GET of
this same URL here (get the enrollment status of a course).

Reply:
I also foresee us wanting to lookup the status of a particular user, which could also be done w/ a GET of this URL, accompanied by the `email` parameter.